code/modules/vending/_vending.dm

```diff
@@ -342,6 +343,23 @@ GLOBAL_LIST_EMPTY(vending_machines_to_restock)
 	if(light_mask && !(machine_stat & BROKEN) && powered())
 		. += emissive_appearance(icon, light_mask, src)
 
+/obj/machinery/vending/examine(mob/user)
+	. = ..()
+	if(refill_canister)
```

Comment:
```suggestion
	if(isnull(refill_canister))
		return // you can add the comment here instead
```

ugly indentation in this proc, let's early return to fix it

Reply:
Can do when I get off work 🤙